### PR TITLE
Engineering guidebook update

### PR DESCRIPTION
**Pull request by Terraspark4941**
## About the PR

Tweaked guidebook stuff in engineering tab, see #22272
Requires #26869 
Ohyeah uhh #13506

**Changelog**

:cl:
- tweak: Updated the engineering section of the guidebook!

### CODE REVIEW

File: Resources/ServerInfo/Guidebook/Engineering/AME.xml [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
+  To assemble an AME, start by wrenching down the controller on the near end of a HV wire. On most stations, there's catwalks to assist with this. From there, start putting down a 3x3 or larger square of AME parts in preparation for construction, making sure to maximize the number of "center" pieces that are surrounded on all 8 sides.
 
-Once this is done, you can use a multitool to convert each AME part into shielding, which should form a finished AME configuration. From there, insert a fuel jar, set the fuel rate to [color=#a4885c]twice the core count or less[/color], and turn on injection.
+  Once this is done, you can use a multitool to convert each AME part into shielding, which should form a finished AME configuration. From there, insert a fuel jar, set the fuel rate to [color=#a4885c]twice the core count or less[/color], and turn on injection. Any more that this ratio will eventually result in the engine [color=#ff0000]overheating and[/color], shortly afterwards, [color=#ff0000]exploding[/color].
```
Comment (Tayrtahn) [inline Resources/ServerInfo/Guidebook/Engineering/AME.xml:16]:
```suggestion
  Once this is done, you can use a multitool to convert each AME part into shielding, which should form a finished AME configuration. From there, insert a fuel jar, set the fuel rate to [color=#a4885c]twice the core count or less[/color], and turn on injection. Any more than this ratio will eventually result in the engine [color=#ff0000]overheating and[/color], shortly afterwards, [color=#ff0000]exploding[/color].
```

File: Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml [hunk 2]
```diff
@@ -1,76 +1,76 @@
...
+  # Airlock Upgrades
+  It is not uncommon for plucky individuals to try and bypass an airlock by meddling with its internal wiring.
 
-Fortunately, certain countermeasures can installed into airlocks to inconvenience any would be trespassers.
+  Fortunately, certain countermeasures can installed into airlocks to inconvenience any would-be trespassers.
 
-## Medium security airlocks
-The most basic form of intrusion deterrence is to install internal steel plating that will prevent access to internal wiring of the airlock. 
+  ## Medium security airlocks
+  The most basic form of intrusion deterrence is to install a steel plating that will prevent access to internal wiring of the airlock.
```
Comment (Tayrtahn) [inline Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml:8]:
```suggestion
  The most basic form of intrusion deterrence is to install steel plating that will prevent access to internal wiring of the airlock.
```

**Comment by Terraspark4941** [comment]
okay gotta sign off for tonight
contributors please find any discrepancies in the diffs

**Comment by Terraspark4941** [comment]
also why is Build failing

File: Resources/ServerInfo/Guidebook/Engineering/NetworkConfigurator.xml [hunk 3]
```diff
@@ -1,39 +1,37 @@
...
-a sink or source.
+  ## Link Mode
+  With link mode, you can click on a device that is capable of device linking and then click on any other device that is either a sink or source.
 
-For example, first clicking on a source like a [color=#a4885c]signal button[/color] and then on sink like a
-[color=#a4885c]small light[/color] opens a UI that displays the source ports on the left side and the sink ports on the right.
+  For example, first clicking on a source, like a [color=#a4885c]signal button[/color], and then on sink, like a [color=#a4885c]small light[/color], opens a UI that displays the source ports on the left side and the sink ports on the right.
 
-Now you can eiter click [color=gray]link defaults[/color] to link the default ports for a source + sink combination or press on a source and then a sink port to connect them.
+  Now you can eiter click [color=gray]Link Defaults[/color] to link the default ports for a source + sink combination, or, press on a source port and then a sink port to connect them.
```
Comment (Tayrtahn) [inline Resources/ServerInfo/Guidebook/Engineering/NetworkConfigurator.xml:28]:
```suggestion
  Now you can either click [color=gray]Link Defaults[/color] to link the default ports for a source + sink combination, or press on a source port and then a sink port to connect them.
```

File: Resources/ServerInfo/Guidebook/Engineering/Power.xml [hunk 4]
```diff
@@ -1,53 +1,53 @@
...
+  ## Power storage
+  Each power storage device presented functions as the transformer for its respective power level (HV, MV, and LV), and also provides a fairly sizable backup battery to help flatten out spikes and dips in power usage.
+  <Box>
+  <GuideEntityEmbed Entity="SMESBasic"/>
+  <GuideEntityEmbed Entity="SubstationBasic"/>
+  <GuideEntityEmbed Entity="APCBasic"/>
+  </Box>
+
+  ## Ramping
+  Contrary to what one might expect from a video game electrical simulation, power is not instantly provided upon request. Generators and batteries take time to ramp up to match the draw imposed on them, which leads to brownouts when there are large changes in current draw all at once; for, example when batteries run out.
```
Comment (Tayrtahn) [inline Resources/ServerInfo/Guidebook/Engineering/Power.xml:28]:
```suggestion
  Contrary to what one might expect from a video game electrical simulation, power is not instantly provided upon request. Generators and batteries take time to ramp up to match the draw imposed on them, which leads to brownouts when there are large changes in current draw all at once; for example when batteries run out.
```

**Review by Tayrtahn — CHANGES_REQUESTED** [review]

File: Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml (hunk 2 above)
Comment (Terraspark4941) [reply Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml:8]: i'm not sure about this one
just steel plating doesn't ring quite right in my head

File: Resources/ServerInfo/Guidebook/Engineering/AccessConfigurator.xml [hunk 5]
```diff
@@ -1,32 +1,33 @@
...
 
-## Repairing damaged ID card readers
-Syndicate agents may attempt to hack access restricted devices through the use of a Cryptographic Sequencer (EMAG). This nefarious tool will completely short out any ID card readers that are attached to the device. 
+  ## Repairing damaged ID card readers
+  Syndicate agents may attempt to hack access restricted devices through the use of a Cryptographic Sequencer (EMAG). This nefarious tool will completely short out any ID card readers that are attached to the device.
 
-Crew members will need to partially de/reconstruct affected devices, and then set appropriate access permissions afterwards using the access configurator, to re-establish access restrictions.
-</Document>
\ No newline at end of file
+  Engineers will need to partially de/reconstruct affected devices, and then set appropriate access permissions afterwards using the access configurator, to re-establish access restrictions.
```
[the remark this answers is not in the document]
Comment (Terraspark4941) [reply Resources/ServerInfo/Guidebook/Engineering/AccessConfigurator.xml:32]: i found that door electronics are apparently broken, so uh this requires #26869 fixed to be merged

File: Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml (hunk 2 above)
Comment (Terraspark4941) [reply Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml]: aha, i think i got something similar (check commit [ded1f45](https://github.com/space-wizards/space-station-14/pull/26851/commits/ded1f45cc52a95e3edbc9ca0424d30291ab4ebf6))

**Review by Tayrtahn — APPROVED** [review]

**Comment by metalgearsloth** [comment]
@Terraspark4941 is this still being worked on.

**Comment by Terraspark4941** [comment]
unfortunately not quite; my exams are in less than a week, so i'm both out
of the loop (on tweaks/updates) and low on time
BUT if i can finalize a few more details, this can be merged
otherwise, it should be frozen until the end of the feature freeze (which
is also my end date for the exams)

**Comment by metalgearsloth** [comment]
okey dokey

**Comment by Terraspark4941** [comment]
mmm i just speedran finishing up this PR, guh
would reopening it add new commits? if so, please do (ready to review i pray)

**Comment by Terraspark4941** [comment]
guh where are the new commits

File: Resources/ServerInfo/Guidebook/Engineering/NetworkConfigurator.xml [hunk 6]
```diff
@@ -1,39 +1,40 @@
 <Document>
-# Network Configurator
-The network configurator allows you to manipulate device lists and link devices together.
-<Box>
-  <GuideEntityEmbed Entity="NetworkConfigurator"/>
-</Box>
-The configurator has two modes: List and Link. You can press [color=gray]Alt+Z[/color] or [color=gray]Alt+Y[/color] to switch between them.
+  # Network Configurator
+  The network configurator allows you to manipulate device lists, link devices together and configure acceses for airlocks through door electronics.
```
Comment (mirrorcult) [inline Resources/ServerInfo/Guidebook/Engineering/NetworkConfigurator.xml:3]:
```suggestion
  The network configurator allows you to manipulate device lists, link devices together and configure accesses for airlocks through door electronics.
```

File: Resources/ServerInfo/Guidebook/Engineering/PortableGenerator.xml [hunk 7]
```diff
@@ -33,10 +33,11 @@
...
-  The (S.U.P.E.R.)P.A.C.M.A.N. is intended for usage by engineering for advanced power scenarios. Bootstrapping the engine, powering departments, and so on.
+  The (S.U.P.E.R.)P.A.C.M.A.N. is intended for usage by engineering for advanced power scenarios. Bootstrapping larger engines, powering departments, and so on.
 
-  The S.U.P.E.R.P.A.C.M.A.N. boasts larger power output and longer runtime at maximum output, but scales down to lower outputs less efficiently.
+  The S.U.P.E.R.P.A.C.M.A.N. boasts a larger power output and longer runtime at maximum output, but scales down to lower outputs less efficiently.
 
-  They connect directly to [color=yellow]MV[/color] or [color=orange]HV[/color] power cables, able to switch between them for flexibility.
+  They connect directly to [color=yellow]MV[/color] or [color=orange]HV[/color] power cables, and are able to switch between them for flexibility.
 
+  The S.U.P.E.R.P.A.C.M.A.N and P.A.C.M.A.N require Plasma sheets and Uranium sheets as fuel, respectively.
```
Comment (mirrorcult) [inline Resources/ServerInfo/Guidebook/Engineering/PortableGenerator.xml:42]:
```suggestion
  The S.U.P.E.R.P.A.C.M.A.N and P.A.C.M.A.N require uranium sheets and plasma sheets as fuel, respectively.
```

its the other way around

File: Resources/ServerInfo/Guidebook/Engineering/Power.xml [hunk 8]
```diff
@@ -1,53 +1,53 @@
...
-2. Pry open the tile that is blocking your access to the cable with a [color=#a4885c]crowbar[/color]. (skip this step if cables aren't hidden)
-3. Equip your trusty [color=#a4885c]Multitool[/color] and click on any cable to see powergrid stats.
-<Box>
-<GuideEntityEmbed Entity="Multitool"/>
-<GuideEntityEmbed Entity="Crowbar"/>
-<GuideEntityEmbed Entity="trayScanner"/>
-</Box>
+  # Power
+
+  SS14 has a fairly in-depth power system through which all devices on the station receive electricity. It's divided into three main powernets; [H]igh [V]oltage, [M]edium [V]oltage, and [L]ow [V]oltage.
```
Comment (mirrorcult) [inline Resources/ServerInfo/Guidebook/Engineering/Power.xml:4]:
```suggestion
  SS14 has a fairly in-depth power system through which all devices on the station receive electricity. It's divided into three main powernets; High Voltage, Medium Voltage, and Low Voltage.
```

just looks weird like that

**Review by mirrorcult — APPROVED** [review]

**Comment by mirrorcult** [comment]
in the future with guidebook PRs pls dont edit the whitespace so much, makes it much more annoying to review the actual changed text

**Pull request merged**